bin/transformatron

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env php
+<?php
+use SBOMinator\Transformatron\Converter;
+use SBOMinator\Transformatron\Exception\ConversionException;
+use SBOMinator\Transformatron\Exception\ValidationException;
+
+const USAGE = <<<'END'
+	Usage: transformatron <path-to-sbom-file> [--format=spdx|cyclonedx] [--target=spdx|cyclonedx]
+	Options:
+		--format=spdx|cyclonedx  Specify the format of the input SBOM file
+		--target=spdx|cyclonedx  Specify the format of the output SBOM file
+		--help                   Display this help message
+		--version                Display the version of this tool
+	END;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$cmd = array_shift($argv);
+$format = 'auto';
+$target = 'auto';
+$path = null;
+foreach ($argv as $arg) {
```

Comment:
This could use getopt, but this was easy enough to write. Happy to convert.

Reply:
This is good for now 💖